CHANGELOG.md

```diff
@@ -109,25 +118,25 @@ continues to improve `tt play` command.
 
 - `tt aeon connect`: add support to connect Aeon database.
 - `tt play`: support of the SSL parameters by using next flags:
-  * `sslkeyfile` - path to a private SSL key file,
-  * `sslcertfile` - path to an SSL certificate file,
-  * `sslcafile` - path to a trusted certificate authorities (CA) file,
-  * `sslciphers` - colon-separated list of SSL cipher suites the connection.
+  - `sslkeyfile` - path to a private SSL key file,
+  - `sslcertfile` - path to an SSL certificate file,
+  - `sslcafile` - path to a trusted certificate authorities (CA) file,
+  - `sslciphers` - colon-separated list of SSL cipher suites the connection.
 - `tt play`: support connection to a target instance by `application` name
   or `application:instance` name.
 - `tt coredump pack`: add options to customize coredump packing:
-  * `-e (--executable)`: specify Tarantool executable path.
-  * `-p (--pid)`: specify PID of the dumped process.
-  * `-t (--time)`: specify time of dump (seconds since the Epoch).
+  - `-e (--executable)`: specify Tarantool executable path.
+  - `-p (--pid)`: specify PID of the dumped process.
+  - `-t (--time)`: specify time of dump (seconds since the Epoch).
```

Comment:
Please, add a separate commit if you want to change a formatting of the whole file.

Reply:
I corrected it, but one line still looks like a new line for some unknown reason.